statistics-service/skaffold.yaml

```diff
@@ -0,0 +1,14 @@
+
+apiVersion: skaffold/v1beta13
+kind: Config
+build:
+  artifacts:
+    - image: keptnsandbox/statistics-service
```

Comment:
please change this to keptn/statistics-service

Reply:
done